rdt/hyper_transformer.py

```diff
@@ -302,17 +305,69 @@ def update_transformers(self, column_name_to_transformer):
             raise Error(self._DETECT_CONFIG_MESSAGE)
 
         for column_name, transformer in column_name_to_transformer.items():
-            current_sdtype = self.field_sdtypes.get(column_name)
-            if current_sdtype and current_sdtype != transformer.get_input_sdtype():
-                warnings.warn(
-                    f'You are assigning a {transformer.get_input_sdtype()} transformer '
-                    f'to a {current_sdtype} column ({column_name}). '
-                    "If the transformer doesn't match the sdtype, it may lead to errors."
-                )
+            if transformer is not None:
+                current_sdtype = self.field_sdtypes.get(column_name)
+                if current_sdtype and current_sdtype != transformer.get_input_sdtype():
+                    warnings.warn(
+                        f'You are assigning a {transformer.get_input_sdtype()} transformer '
+                        f'to a {current_sdtype} column ({column_name}). '
+                        "If the transformer doesn't match the sdtype, it may lead to errors."
+                    )
 
             self.field_transformers[column_name] = transformer
             self._provided_field_transformers[column_name] = transformer
 
+    def remove_transformers(self, column_names):
+        """Remove transformers for given columns.
+
+        This will remove the transformer for a given column name and this will not be
+        transformed.
+
+        Args:
+            column_names (list):
+                List of columns to remove the transformers for.
+        """
+        unknown_columns = []
+        for column_name in column_names:
+            if column_name not in self.field_transformers:
+                unknown_columns.append(column_name)
+            else:
+                self.field_transformers[column_name] = None
+                self._provided_field_transformers[column_name] = None
+
+        if self._fitted:
+            warnings.warn(self._REFIT_MESSAGE)
+
+        if unknown_columns:
+            raise Error(
```

Comment:
Should we be raising the error after the fact, or should all changes be avoided if there are any unknown columns?  @npatki Basically if some columns are valid, should those still be changed or should the method fail completely?

Reply:
Yeah the entire think should error. That way the user can fix it and re-pass the entire list.